pandas/_libs/parsers.pyx

```diff
@@ -178,13 +178,13 @@ cdef extern from "parser/tokenizer.h":
         char thousands
 
         int header # Boolean: 1: has header, 0: no header
-        int header_start # header row start
-        int header_end # header row end
+        ssize_t header_start # header row start
+        ssize_t header_end # header row end
```

Comment:
Also, why `ssize_t` here (and not `size_t` like above)?  We should either use one or the other IMO with these changes.  I think either would work since technically, all of these fields must be >= 0.